tls/s2n_config.h

```diff
@@ -30,18 +30,28 @@
 struct s2n_cipher_preferences;
 
 struct s2n_config {
+    /* The following bitfield flags are used in SAW proofs. The position of
+     * these flags is important to SAW, so make sure that any new flags are
+     * added after these ones.
```

Comment:
Could you just improve this comment with a sentence explaining why "the position of these flags is important to SAW"?